### PR TITLE
Make Module.transform configurable from evaluate

### PR DESCRIPTION
**Pull request by giuseppeg**
Replacement of #246

### CODE REVIEW

File: src/babel/evaluate.js [hunk 1]
```diff
@@ -58,7 +58,8 @@ const resolve = (path, t, requirements) => {
 module.exports = function evaluate(
   path /* : any */,
   t /* : any */,
-  filename /* : string */
+  filename /* : string */,
+  transform /* : ?(text: string) => { code: string } */ = null
```
Comment (satya164) [inline src/babel/evaluate.js]: Thanks for the PR!

Just a `nit`, but can you extract the `defaultTransform` function to top of this function and use default parameter here? I think it'll be simpler.

Comment (giuseppeg) [reply src/babel/evaluate.js]: good idea thanks, done :)

Comment (giuseppeg) [reply src/babel/evaluate.js]: let me know if you prefer `transform` over `transformModule`

**Comment by giuseppeg** [comment]
awesome, thank you guys! :)

**Pull request merged**